docs/reference.md

```diff
@@ -303,3 +303,13 @@ spec:
             name: a-different-input
             namespace: default
 ```
+
+### Deletion Behavior
+
+Symphonies are high-level resources designed to always converge, even in the face of rare split-brain states.
+
+Force deleting namespaces leaves resources in a strange state in which they exist but cannot be updated.
+Symphonies recover from this state by carefully recreating the namespace and forcibly removing internal finalizers.
+Because of this it's possible that managed resources will not be cleaned up if they exist outside of the orphaned namespace.
+Worst case, managed resources might be recreated by the Eno reconciler if it outpaces kube-controller-manager's namespace controller.
```

Comment:
If the resources are in the same namespace as the symphony, yes - the KCM namespace controller will delete them, even though Eno won't.

Symphonies are really intended to be used as a namespace-scoped concept (that's our internal use of them anyway). So I think we should clarify that in the docs later on and probably make some behavior changes to avoid getting into situations where resources could be orphaned.